client/src/components/Avatar.js

```diff
@@ -19,39 +20,79 @@ const UploadImage = styled.button`
   width: 25px;
   margin-left: 35px;
   position: absolute;
+  align-items: center;
   justify-content: center;
+  background: url(${cameraIcon}) no-repeat;
+  background-position: center;
   border-radius: 50%;
   border: none;
   background-color: white;
   outline: none;
+  cursor: pointer;
   &:active {
     transform: translateY(1px) scale(0.9);
     box-shadow: 0px 5px 10px rgba(0, 0, 0, 20%);
   }
 `;
 
+const ImageInput = styled.input`
+  display: flex;
+  height: 20px;
+  width: 20px;
+  opacity: 0%;
+   position relative;
+   border-radius: 50%;
+  border: none;
+  focus {
+    outline: none;
+  }
+  overflow: hidden;
+`;
+
```

Comment:
Looks like prettier didn't do a good job here. Maybe there is an issue in this file?

Reply:
I don't know what happened, because my prettier is also active, but no problem is displayed... fixed it myself 🤔